### PR TITLE
SEO-175190 Xamarin Redirect

### PR DESCRIPTION
**Pull request by Yvone-Atieno**
Hi @Aishwarya-Ganesan,

Have changed Page with Redirect links.

Regards,
Yvone.

### CODE REVIEW

File: Xamarin/DataSource/DataSource-GettingStarted.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ DataSource is a non UI component that consumes raw data and processes data opera
 
 ### Assembly deployment
 
-After installing [Essential Studio for Xamarin](https://www.syncfusion.com/downloads/xamarin), you can find all the required assemblies in the installation folders, {Syncfusion Essential Studio Installed location} \Essential Studio\\{Version #}\Xamarin\lib.
+After installing [Essential Studio for Xamarin](https://www.syncfusion.com/account/manage-trials/downloads), you can find all the required assemblies in the installation folders, {Syncfusion Essential Studio Installed location} \Essential Studio\\{Version #}\Xamarin\lib.
```
Comment (MallikaRK) [inline Xamarin/DataSource/DataSource-GettingStarted.md:22]: image
revert your changes. check incognito window for this.

Comment (MallikaRK) [reply Xamarin/DataSource/DataSource-GettingStarted.md:22]: revert your changes in all places, you made this.

**Review by MallikaRK — CHANGES_REQUESTED** [review]
Commented

File: Xamarin/DataSource/DataSource-GettingStarted.md (hunk 1 above)
Comment (Yvone-Atieno) [reply Xamarin/DataSource/DataSource-GettingStarted.md:22]: reverted changes

**Review by MallikaRK — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are fine.

**Pull request merged**